interface/patient_file/report/patient_report.php

```diff
@@ -686,6 +788,11 @@ function issueClick(issue) {
     });
 }
 
+function formSubmit() {
+  if (typeof(top.restoreSession) == "function") {
```

Comment:
I didn't think that was an option here?  Better to show the error than to silently ignore it.